packages/wagmi/package.json

```diff
@@ -44,12 +44,12 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "typescript": "^4.9.5",
-    "wagmi": "^0.11.6"
+    "wagmi": ">=0.9.2 <=0.11.6"
   },
   "peerDependencies": {
     "@lens-protocol/react": "workspace:*",
     "ethers": "^5.7.2",
-    "wagmi": "^0.11.5"
+    "wagmi": ">=0.9.2 <=0.11.6"
```

Comment:
We should have the range only here inside `peerDependencies`, all the other places are considered internal and can use specific versions (with or without `^` prefix).

Reply:
ok 👌